### PR TITLE
Add fan on/off buttons

### PR DESCRIPTION
**Pull request by android444**
Add two butons.
I have added relevant entries in language files - no translations.

### CODE REVIEW

File: Marlin/src/lcd/language/language_it.h [hunk 1]
```diff
@@ -251,6 +251,8 @@ namespace Language_it {
   PROGMEM Language_Str MSG_CONTROLLER_FAN_AUTO_ON          = _UxGT("Modo autom.");
   PROGMEM Language_Str MSG_CONTROLLER_FAN_SPEED            = _UxGT("Vel. attivo");
   PROGMEM Language_Str MSG_CONTROLLER_FAN_DURATION         = _UxGT("Tempo inattivo");
+  PROGMEM Language_Str MSG_FAN_ON                          = _UxGT("Fan on");
```
Comment (GMagician) [inline Marlin/src/lcd/language/language_it.h:254]: If it's meant to be an action:
Fan On=Accendi ventola
Fan off=Spegni ventola
If it's meant to be a status:
Fan On=Ventola accesa
Fan off=Ventola spenta

Comment (thinkyhead) [reply Marlin/src/lcd/language/language_it.h:254]: GitHub now allows you to add "change requests" directly when viewing a file. If the author accepts the change request then it is applied as a new commit from you.

**Comment by thinkyhead** [comment]
Your repository went away so this is broken and can't be edited. Please fix and re-submit.

And be sure to fix…
```cpp
if !READ(
```

**Comment by android444** [comment]
Sorry. My mistake.

**Pull request closed without merging**